WME_Image_Overlays.user.js

```diff
@@ -77,12 +53,7 @@ function init(e) {
   setTranslations(io_strings);
   applyStyles();
 
-  var tab = addTab();
-
-  // Deal with measuring unit switches between metric and imperial
```

Comment:
This is no longer needed. Changing between modes does not cause WME to completely refresh. Further, the new API keeps track of userscript tabs,